src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -420,6 +421,10 @@ public void setAvd(String avd) {
         this.avd = avd;
     }
 
+    public String getAvdArgs() { return avdArgs; }
```

Comment:
Just thinking out loud: If we're handling multiple arguments, do we need to put all of the avdargs into an array and then get those? Perhaps this would allow more specificity when selecting env variables from CI to represent these args. 

On CI we may need to handle `&` being an argument as well, not sure if needed with Appium setup though.